tools/parse-release-notes

```diff
@@ -32,7 +32,9 @@ const formatNotes = ({ notes, language, production, platformName }) => {
   const formattedNotes = notes
     .map(note => {
       const localizedNote = note[language] || note[DEFAULT_LANGUAGE]
-      return production ? `* ${localizedNote}` : `* [ ${note.issue_key} ] ${localizedNote}`
+      // Double quotes make mattermost status alerts fail
+      const escapedNote = localizedNote.replace(/"/g, '\'')
```

Comment:
Here is a more robust way: 
```
console.log(JSON.stringify('This is a test release note with quotes: " <-', null, 2).slice(1, -1))
```

It prints:

```
This is a test release note with quotes: \" <-
```

Reply:
This also covers backslashes for example.

Reply:
Oh right, in that case it would have been: `console.log(JSON.stringify(JSON.stringify('This is a test release note with quotes: " <-', null, 2).slice(1, -1)).slice(1, -1))`